pipeline.py

```diff
@@ -11,7 +11,18 @@
 import oneaday_filter
 import result_formatter
 import scraper_connection
-from petrarch import petrarch
+#from petrarch2 import petrarch2
+
+server_details, file_details, petrarch_version = utilities.parse_config('PHOX_config.ini')
```

Comment:
My initial reaction is that I don't like pulling this out of a function. What's your reasoning behind that?